src/lua/init.c

```diff
@@ -53,6 +53,7 @@
 #include "lua/utils.h"
 #include "third_party/lua-cjson/lua_cjson.h"
 #include "third_party/lua-yaml/lyaml.h"
+#include "extra/lua-zlib/lua_zlib.h"
```

Comment:
I think a file that is needed during tarantool build should not be placed in extra/. I think ideally it should be part of our lua-zlib distribution.